builtin/logical/nomad/path_config.go

```diff
@@ -0,0 +1,104 @@
+package nomad
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfigAccess() *framework.Path {
+	return &framework.Path{
+		Pattern: "config/access",
+		Fields: map[string]*framework.FieldSchema{
+			"address": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Nomad server address",
+			},
+
+			"scheme": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "URI scheme for the Nomad address",
+
+				// https would be a better default but Consul on its own
+				// defaults to HTTP access, and when HTTPS is enabled it
+				// disables HTTP, so there isn't really any harm done here.
+				Default: "http",
+			},
+
+			"token": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Token for API calls",
+			},
+		},
+
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.ReadOperation:   pathConfigAccessRead,
+			logical.UpdateOperation: pathConfigAccessWrite,
+		},
+	}
+}
+
+func readConfigAccess(storage logical.Storage) (*accessConfig, error, error) {
+	entry, err := storage.Get("config/access")
+	if err != nil {
+		return nil, nil, err
+	}
+	if entry == nil {
+		return nil, fmt.Errorf(
```

Comment:
Just return `nil`, `nil`  and raise the user error to the user.

Reply:
Fixed in dcaec0a880682c04159029fc78296064c7a9db6d